supportutilsPluginSUSEPublicCloud/supportconfig/plugins/suse_public_cloud

```diff
@@ -0,0 +1,180 @@
+#!/bin/bash
+
+# Supportconfig Plugin for SUSE Public Cloud Images
+# 
+# Copyright (C) 2016 SUSE LLC, Robert Schweikert <rjschwei@suse.com>
+# 
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+# 
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor,
+# Boston, MA  02110-1301, USA.
+
+# Gathers important troubleshooting information about
+# SUSE Public Cloud Instances
+
+SVER='1.0'
+RCFILE="/usr/lib/supportconfig/resources/scplugin.rc"
```

Comment:
It's good to stay consistent with the quoting in bash, the policy here are " quotes, I would avoid ' quotes